common/test/acceptance/tests/lms/test_lms_completion_aggregator.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+"""
+End-to-end tests for the LMS that test the completion aggregates as the learner progresses in the course
+"""
+from contextlib import contextmanager
+
+from ...fixtures.course import CourseFixture, XBlockFixtureDesc
+from ...pages.common.logout import LogoutPage
+from ...pages.lms.completion_aggregate import CompletionAggregatePage
+from ...pages.lms.courseware import CoursewarePage
+from ...pages.lms.instructor_dashboard import InstructorDashboardPage
+from ...pages.lms.problem import ProblemPage
+from ...pages.studio.overview import CourseOutlinePage as StudioCourseOutlinePage
+from ...pages.studio.utils import drag
```

Comment:
nit: could you use absolute imports? checking at the other tests code, they are all using absolute imports, and I think it makes sense to use the same style for this ones. Also it is a bit more clear when you are reading it, to see where the import is coming from